frontend/src/components/suite/LogMessages.styles.js

```diff
@@ -86,3 +86,46 @@ export const InfoLevel = styled.td`
         margin: 0 10px;
     }
 `;
+
+export const TableWrapper = styled.div`
+    max-width: calc(100% + 40px);
+    position: relative;
+    margin: 0 -20px;
+    &::before {
+        position: absolute;
+        top: 0;
+        left: 0;
+        content: '';
+        width: 20px;
+        height: 100%;
+        background-image: linear-gradient(
+            to right,
+            white,
+            rgba(255, 255, 255, 0.001)
+        );
+        z-index: 1;
```

Comment:
since multiple z-indexes are starting to be used I'd see it wise to collect these with info about the levels used and where to avoid future collisions